components/phpunit_bridge.rst

```diff
@@ -0,0 +1,129 @@
+.. index::
+   single: PHPUnitBridge
+   single: Components; PHPUnitBridge
+
+The PHPUnit Bridge Component
+=============================
+
+    The PHPUnit Bridge component provides utilities to report trigerred errors,
+    legacy tests and usage of deprecated code.
```

Comment:
It provides a lot more than just the deprecation helper:

 * A [bootstrap file](https://github.com/symfony/phpunit-bridge/blob/master/bootstrap.php) to configure the env correctly for testing (file is automatically loaded by Composer during tests)
 * A [ClockMock](https://github.com/symfony/phpunit-bridge/blob/master/ClockMock.php) for mocking time in time sensitive tests
 * A [SymfonyTestListener](https://github.com/symfony/phpunit-bridge/blob/master/SymfonyTestsListener.php) that (a) automatically registers the ClockMock if the test has the group `time-sensitive` and (b) comes with a Skipped feature to only test the previously skipped tests. I don't  think (b) is very usefull for others though (it's used on appveyor, where tests are run twice, onces with min settings and one with max, during the second run only skipped tests of the first run are executed).

Please note that the SymfonyTestListener is also automatically loaded.